quicktest_test.go

```diff
@@ -149,6 +149,18 @@ func TestCRunPanic(t *testing.T) {
 	assertBool(t, run, true)
 }
 
+func TestCRunCleanup(t *testing.T) {
+	c := qt.New(&testingT{})
+	outerClean := 0
+	innerClean := 0
+	c.AddCleanup(func() { outerClean++ })
+	c.Run("x", func(c *qt.C) {
+		c.AddCleanup(func() { innerClean++ })
+	})
+	c.Assert(innerClean, qt.Equals, 1)
+	c.Assert(outerClean, qt.Equals, 0)
+}
+
 func checkResult(t *testing.T, ok bool, got, want string) {
 	if want != "" {
 		assertPrefix(t, got, "\n"+want)
```

Comment:
We should add tests for AddCleanup and Cleanup, as they seem to be exercised only indirectly.

Reply:
done